pages/accountLists/GetDashboard.graphql

```diff
@@ -29,3 +29,21 @@ query GetDashboard($accountListId: ID!) {
     }
   }
 }
+
+query GetQuestions($accountListId: ID!, $completed: Boolean = false) {
```

Comment:
This is temporary-- just to test the query.  I will remove this before merging the PR.